src/app/mod.rs

```diff
@@ -23,28 +23,34 @@ impl App {
         &self,
         ids: Vec<String>,
         names: Vec<String>,
+        ignore_cache: Option<bool>,
```

Comment:
I don't think it makes sense to make this argument an Option when there's no different between None and Some(false) - it can just be a boolean directly